packages/plugin-session-replay-browser/test/session-replay.test.ts

```diff
@@ -1,62 +1,35 @@
-/* eslint-disable jest/expect-expect */
-import * as AnalyticsClientCommon from '@amplitude/analytics-client-common';
 import { CookieStorage, FetchTransport } from '@amplitude/analytics-client-common';
```

Comment:
Will importing `@amplitude/analytics-client-common` create circular dependencies? Should we mock these instead?

Reply:
The session replay plugin isn't imported anywhere, definitely not to analytics-client-common, so I don't think this should be an issue! But actually I'll remove the use of this anyways, these values aren't used by the plugin anyways